stories/Molecules/DownloadRow/DownloadRow.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './downloadrow.scss';
+import { Checkbox } from '../../Atom/Form/Checkbox/Checkbox';
+
+export const DownloadRow = ({
+  title, subtitle, value, ...args
+}) => (
+  <div className="selectbox-container">
+    <div className="selectbox-container__data">
+      <label htmlFor="title" name="title" className="selectbox-container__data--headertext">{title}</label>
+      <div className="selectbox-container__data--subtext">{subtitle}</div>
```

Comment:
@pallavikadam-srijan We can combine it like <div>{title} <span>{subtitle}</span></div>